bin/extract.js

```diff
@@ -12,7 +12,8 @@ var options = cli.parse({
 	dir: ['d', 'Directory path you would like to extract strings from', 'dir', process.env.PWD],
 	output: ['o', 'Directory path you would like to save extracted strings', 'dir', process.env.PWD],
 	format: ['f', 'Output format', ['json', 'pot'], 'json'],
-	merge: ['m', 'Merge options, true to merge, false to replace, clean to remove unused keys while merging', [true, false, 'clean'], true]
+	replace: ['r', 'Replace the content of the file if it exists (merging by default)'],
```

Comment:
Maybe it's just me, but the name `replace` kind of makes me think it's not an option for enabling merge, but rather a if it can replace an existing file or not?